### DISCUSSION TITLE
Hardware requirements

### DISCUSSION THREAD

**Discussion by cedric20005** [netbox-community/netbox#14138] [Q&A]
Are there any best practice hardware requirements to run NetBox On-Prem?

**Comment by mtinberg** [comment]
I don't think there are any special recommendations, its not a particularly heavy webapp and any Django sizing guides would be fine.  It is most likely that this will run just fine on the resources available in a current cell phone, 4G RAM and a few CPU cores. If you need more resources then scaling is fairly straight forward, more CPU cores and RAM for concurrent workers handling requests either with bigger VMs or horizontally scaling behind a load balancer, with a persistent shared Django session database, or a bigger PostgreSQL if you have high query/transaction counts for some reason.




—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: cedric ***@***.***>
Sent: Monday, October 30, 2023 6:11 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] Hardware requirements (Discussion #14138)


Are there any best practice hardware requirements to run NetBox On-Prem?
